src/main/java/com/google/api/codegen/viewmodel/ImportSectionView.java

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.viewmodel;
+
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+
+@AutoValue
+public abstract class ImportSectionView {
+  public abstract List<ImportTypeView> typeImports();
+
+  public abstract List<ImportFileView> standardImports();
+
+  public abstract List<ImportFileView> externalImports();
+
+  public abstract List<ImportFileView> appImports();
+
+  public abstract List<ImportFileView> serviceImports();
+
+  public static Builder newBuilder() {
+    return new AutoValue_ImportSectionView.Builder()
+        .typeImports(ImmutableList.<ImportTypeView>of())
+        .standardImports(ImmutableList.<ImportFileView>of())
+        .externalImports(ImmutableList.<ImportFileView>of())
+        .appImports(ImmutableList.<ImportFileView>of())
```

Comment:
'protoImports' is applicable to other languages too and it clearly tells what we're importing.